### PR TITLE
Feature/cc 6 create readme

### PR DESCRIPTION
**Pull request by Lexander1108**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+> npm install
+```
+
+### Run
+``` shell
+> npm start
+```
+
+### Docker
+### You can use [docker desktop](https://www.docker.com/products/docker-desktop) to start the project
```
Comment (DrSmile444) [inline README.md:36]: It should be a plain text, not a header

File: README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+2. [FAQ and help](#faq-and-help)
+3. [Credits](#credits)
+
+## Quick start
+
+### Requirements
+* [node v13](https://nodejs.org/dist/v13.12.0/)
+* [postgres v12](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads)
+
+### Installiation
```
Comment (DrSmile444) [inline README.md:25]: "Installation" would be correct. Don't forget to fix the "Table of Contents" link

File: README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+## The goal of this project is to make Cherkasy cleaner and better.
+
+Our product divided into three main projects: Telegram Bot, Backend, and Frontend. Every project is needed to solve a specific problem. Backend manages database and endpoints that allow to:
+* add new cases (from Bot);
+* send all or specific cases (to Frontend);
+* change the status of a case (on Frontend).
+
+## Table of Contents
+1. [Quick start](#qick-start)
```
Comment (DrSmile444) [inline README.md:11]: #qick-start

Link doesn't works

File: README.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+``` shell
+> npm run docker
+```
+
+To insert mock data into the base you need to run
+``` shell
+> npm run docker:seed
+```
+
+To run migration use this command
```
Comment (DrSmile444) [inline README.md:53]: Migration is more important than seed, so move it before the *seed* command

File: README.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+``` shell
+> npm run docker:migrate
+```
+
+## FAQ and help
+If you got stuck or need some help, contact me
+* Telegram: `@lexander_b1108`
+* Gmail: `alexander.kretov99@gmail.com`
```
Comment (DrSmile444) [inline README.md:61]: Use more interactive links:

```md
[alexander.kretov99@gmail.com](mailto:alexander.kretov99@gmail.com?subject=[GitHub]%20CheClean)
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+To run migration use this command
+``` shell
+> npm run docker:migrate
+```
+
+## FAQ and help
+If you got stuck or need some help, contact me
+* Telegram: `@lexander_b1108`
```
Comment (DrSmile444) [inline README.md:60]: Use more interactive links:

`[@lexander_b1108](https://t.me/lexander_b1108)`

File: README.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+* Gmail: `alexander.kretov99@gmail.com`
+
+## Credits
+Made by Master of Code studensts
```
Comment (DrSmile444) [inline README.md:64]: You have typo in the 'students' word

File: README.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
+# CheClean ![](docs/CheClean-logo.png)
```
Comment (DrSmile444) [inline README.md:1]: You forgot to add **alt** image text. All images should have 'alt' text that will display when an image isn't available.

So, in this case you're using the CheClean Logo image and you can use **che-clean-logo** alt text

File: README.md (hunk 1 above)
Comment (Lexander1108) [reply README.md:36]: Okey, fixied it

File: README.md (hunk 3 above)
Comment (Lexander1108) [reply README.md:11]: Thank you, my bad

File: README.md (hunk 2 above)
Comment (Lexander1108) [reply README.md:25]: Fixed

File: README.md (hunk 8 above)
Comment (Lexander1108) [reply README.md]: Fixed this

File: README.md (hunk 7 above)
Comment (Lexander1108) [reply README.md]: My bad, thank you

File: README.md (hunk 6 above)
Comment (Lexander1108) [reply README.md]: Okay, thanks

File: README.md (hunk 5 above)
Comment (Lexander1108) [reply README.md]: Got it

File: README.md (hunk 4 above)
Comment (Lexander1108) [reply README.md]: Thank you

**Pull request merged**